Aquarius/Src/Aquarius/Events/MouseEvent.h

```diff
@@ -0,0 +1,135 @@
+#pragma once
+
+#include "Event.h"
+
+#include <GLFW/glfw3.h>
+
+
+namespace Aquarius {
+
+    class MouseMovedEvent : public Event
+    {
+    public:
+        MouseMovedEvent(float xCoordinate, float yCoordinate)
+                : m_XCoordinate(xCoordinate), m_YCoordinate(yCoordinate) {}
+
+        EVENT_TYPE(MouseMovedEvent)
+
+        void logEvent()
+        {
+            AQ_CORE_TRACE("MouseMovedEvent: %v, %v", m_XCoordinate, m_YCoordinate);
+        }
+
+        float getXCoordinate()
+        {
+            return m_XCoordinate;
+        }
+
+        float getYCoordinate()
+        {
+            return m_YCoordinate;
+        }
+
+    private:
+        float m_XCoordinate;
+        float m_YCoordinate;
+    };
+
+    class MouseScrolledEvent : public Event
+    {
+    public:
+        MouseScrolledEvent(float xCoordinate, float yCoordinate)
+                : m_XCoordinate(xCoordinate), m_YCoordinate(yCoordinate) {}
+
+        EVENT_TYPE(MouseScrolledEvent)
+
+        void logEvent()
+        {
+            AQ_CORE_TRACE("MouseScrolledEvent: %v, %v", m_XCoordinate, m_YCoordinate);
+        }
+
+        float getXCoordinate()
+        {
+            return m_XCoordinate;
+        }
+
+        float getYCoordinate()
+        {
+            return m_YCoordinate;
+        }
+
+    private:
+        float m_XCoordinate;
+        float m_YCoordinate;
+    };
```

Comment:
Maybe make the member names better reflect a mouse scroll, this is a copy+paste of the MouseMovedEvent